src/Auth/ReadFromTextFileCredentials.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace YdbPlatform\Ydb\Auth;
+
+class ReadFromTextFileCredentials extends Auth
+{
+
+    protected $fileName;
+    protected $readInterval;
+
+    /**
+     * @param string $fileName
+     * @param int $readInterval
+     */
+    public function __construct(string $fileName = "token.json", int $readInterval = 600)
```

Comment:
I suggest set readInteval to one minute instead of 10 minutes.

1 and 10 minutes - similar to computer systems (for fase operations), but different for human - if you will need to see behaviour in real system.